content/docs/Configuration/jobs.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Packit Service jobs configuration 
+date: 2023-07-03
+weight: 1
+---
+
+# Packit Service jobs configuration
+
+Packit Service doesn't have any web interface, but its behaviour can be configured via the config file 
+generally described [here](/docs/configuration).
+
+The tasks Packit Service should do should be defined in the `jobs` section. The section is a list of dictionaries:
+```yaml
+jobs:
+- {key: value}
+- {}
+```
+
+When there is an action (pull request, commit, etc.) in the upstream repository, Packit Service takes the config 
+file from the relevant git ref (release tag for release, PR head for pull requests, related dist-git commit for downstream jobs).
+Then, it tries to find a matching job in the `jobs` sections for the particular event (e.g. for pull request, jobs with
+`pull_request` trigger are matched).
+
+If there is no `jobs` section in the configuration file, jobs default to:
```

Comment:
Off-topic, but would be interesting to check if anyone is actually using the default value.

Also, we can add  koji and bodhi into the default .

Reply:
good suggestions! I will check and we can discuss

Reply:
Looks like not many users:
```
Affected repos (3): 
https://github.com/varlink/libvarlink
https://github.com/skupperproject/skupper-router
https://github.com/candlepin/virt-who
```